sky/utils/schemas.py

```diff
@@ -83,7 +83,10 @@ def get_storage_schema():
                 'type': 'string',
             },
             'source': {
-                'type': 'string',
+                'type': ['string', 'array'],
```

Comment:
how do you specify a Union[string, array of strings]?

Reply:
I believe it's [type1, type2] [source](https://cswr.github.io/JsonSchema/spec/multiple_types/)

Reply:
In that case, is it possible to replace `array` with:

```
{
  "type": "array",
  "items": {
    "type": "string"
  }
}
```

Reply:
@romilbhardwaj Double check this one, does this work?